package.json

```diff
@@ -101,6 +101,7 @@
     "changelog": "changelog",
     "build": "npm run locales && npm run lint && jpm xpi",
     "test": "npm run locales && jpm run -b nightly -p Testing",
+    "test-nocopy": "npm run locales && jpm run -b nightly -p Testing --no-copy",
```

Comment:
I don't think we need a whole new script in `package.json` for this.  Please revert this change.